packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -25,9 +25,9 @@ export class VaTelephone {
   @Prop() contact!: string;
 
   /**
-   * Optional phone number extension
+   * Optional numeric string phone number extension
    */
-  @Prop() extension?: number;
+  @Prop() extension?: string;
```

Comment:
The `contact` prop (for the main phone number) is also type `string`: 

https://github.com/department-of-veterans-affairs/component-library/blob/12df9ae4d584652cec9dfe3ef3cbcc0540831395/packages/web-components/src/components/va-telephone/va-telephone.tsx#L22-L25